### DISCUSSION TITLE
Option to show x-axis value at the bottom of the graph when Y-axis have negative values

### DISCUSSION THREAD

**Discussion by sandeep-synclovis** [Abhinandan-Kushwaha/react-native-gifted-charts#838] [Ideas]
Hi,

Currently, If Y-axis have negative values x-axis is coming at the centre of the graph. Please refer below shared image:

<img width="275" alt="Screenshot 2024-09-16 at 1 53 34 PM" src="https://github.com/user-attachments/assets/72a53d23-7bb6-4ea2-aaea-de5f6d5b9d3f">

If possible can we get the option to show the x-axis at the bottom of the graph, like the below shared image?

<img width="277" alt="Screenshot 2024-09-16 at 1 55 05 PM" src="https://github.com/user-attachments/assets/9ec063df-c922-417f-8da9-32d08b3e0ecd">

**Comment by Abhinandan-Kushwaha** [comment]
@sandeep-synclovis Try using the `xAxisLabelsVerticalShift` prop.

**Comment by sandeep-synclovis** [comment]
Hi, I tried but in this case x-axis where the value is negative shifted upwards and for others, it shifted downwards. I made autoShiftLable false but still, it's the same.


<img width="280" alt="Screenshot 2024-09-16 at 4 58 32 PM" src="https://github.com/user-attachments/assets/24b3eea4-cabb-490a-bdf9-c5ee577470e9">

**Comment by Abhinandan-Kushwaha** [comment]
@sandeep-synclovis Set the prop `autoShiftLabels={false}`

**Comment by sandeep-synclovis** [comment]
Hi, I already did autoShiftLabels={false}, but still the same, I mentioned this in the above comment also.

**Comment by Abhinandan-Kushwaha** [comment]
Hi @sandeep-synclovis This has been fixed from versions `1.4.40` onwards. 🎉
See https://gifted-charts.web.app/barchart/#xAxisShift

Please use the latest version of the library.
